### DISCUSSION TITLE
How to import data from Suunto Quest without Moveslink

### DISCUSSION THREAD

**Discussion by ZeroDayOwl** [mytourbook/mytourbook#1367] [Q&A]
Hello! I’m attempting to import data from a Suunto Quest watch into MTB. Although I don’t have Moveslink and haven’t used it before, I do have the Movestick Mini dongle. After installing the drivers from Moveslink’s old website, the Movestick Mini is properly detected on the PC. Unfortunately, without Moveslink software installed, I can’t directly sync the watch data to my PC for MTB. And the watch doesn't detect the PC either (I'm guessing it's because the Moveslink software isn't running on the PC). I’ve tried using the Serial Port options in MTB trying to import the data, but Quest doesn’t appear on the list. Additionally, since my watch lacks Bluetooth (BT), I can’t connect it to the Suunto App. 

Is there an alternative method to import the data from my watch to the PC? Thank you in advance for any assistance!

**Comment by stvzldn** [comment]
I don't have this watch, but have you asked on the Sunnto Community Forum for help?
https://forum.suunto.com/
It may be that uploading to strava will let you then download data back in an MTB usable format.
Saw this: https://forum.suunto.com/topic/9332/getting-training-data-from-strava

**Reply by ZeroDayOwl** [reply]
Yeah I have checked the forums, there's not much of help there regarding this. Unfortunately, this 2012 watch doesn’t have Bluetooth support, so you can’t directly connect it to the Suunto app or upload files to Strava. To get the data, you’ll need to extract it from the watch first.

**Comment by FJBDev** [comment]
I added the import for Suunto Quest files in the 20.8 release of MT

See more details [here](https://github.com/mytourbook/mytourbook/issues/825)

**Reply by ZeroDayOwl** [reply]
Hi, I have checked your PR regarding adding the import feature in MTB. However, as I assume, that works only when you are trying to import the Quest XML files from local computer to MTB. Unfortunately, I don't have the XML files locally and was wondering if there were any ways to use MTB as an interface to import data from the watch itself, if possible.

**Reply by FJBDev** [reply]
> Hi, I have checked your PR regarding adding the import feature in MTB. However, as I assume, that works only when you are trying to import the Quest XML files from local computer to MTB. 

Correct, MT will import the XML files from any Suunto Quest but that is assuming that the files exist on disk.

>Unfortunately, I don't have the XML files locally and was wondering if there were any ways to use MTB as an interface to import data from the watch itself, if possible.

MT does not offer a way to retrieve the files from the watch but I would suggest to try to get MovesLink installed although it seems to have been deprecated a while back

**Reply by ZeroDayOwl** [reply]
> MT does not offer a way to retrieve the files from the watch but I would suggest to try to get MovesLink installed although it seems to have been deprecated a while back

That's right. I tried looking for Moveslink but the one available was Moveslink 2, which doesn't support the ANT protocol on the Quest watch. It only supports watches with USB connections directly from it.

**Comment by mff73** [comment]
@FJBDev, I think the OP asked to extract files from the watch itself.

@ZeroDayOwl, why don't you install Moveslink?
Not sure it will work again on new windows, but who knows.

**Reply by ZeroDayOwl** [reply]
Yep you're right.

I tried looking for the original Moveslink software, but couldn't find any from trustable sources. Do you know of any?

Also, not sure if it's gonna work unless you have a movescount account.

**Reply by FJBDev** [reply]
> I tried looking for the original Moveslink software, but couldn't find any from trustable sources. Do you know of any?
> 

A quick Google search lead me to [this thread.](https://forum.suunto.com/post/52722)

I downloaded the file mentioned and installed it on Windows 10 and it seems to work.
I hope this helps!

![image](https://github.com/mytourbook/mytourbook/assets/25235665/758c319c-12ef-4272-8f89-f86d7ed9c3e5)

**Reply by ZeroDayOwl** [reply]
Hey @FJBDev thanks for the link, it installs and runs correctly on my PC too. However, it doesn't support Quest watch :(

**Reply by FJBDev** [reply]
> Hey @FJBDev thanks for the link, it installs and runs correctly on my PC too. However, it doesn't support Quest watch :(

@ZeroDayOwl Bummer.... Did you try to contact Suunto to ask them what they recommend in order to be able to retrieve the files from your watch ?

**Comment by FJBDev** [comment]
@ZeroDayOwl You could also try [OpenAmbit ](https://github.com/openambitproject/openambit)as maybe it supports retrieving files from a Quest ?

**Reply by ZeroDayOwl** [reply]
Unfortunately it only supports Ambit series. ([Supported Devices](https://github.com/openambitproject/openambit/wiki/Supported-devices))
